common/messages/messages_en.go

```diff
@@ -74,7 +74,7 @@ var messagesEn = map[string]util.Message{
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
-		Description: "No volume access point could not be found for the specified volume ID '%s' and access point ID '%s' or VPC ID %s",
+		Description: "No volume access point could not be found for the specified volume ID '%s' and access point ID '%s'",
```

Comment:
This line doesn't looks to be correct. Instead we can have Volume access point could not be found for the specified volume ID '%s' with access point ID '%s'

Reply:
I can do that but this standard message was followed to keep consistency between block library.
"	"VolumeAttachFindFailed": {
		Code:        VolumeAttachFindFailed,

		Description: "No volume attachment could be found for the specified volume ID '%s' and instance ID '%s'.",
		Type:        util.VolumeAttachFindFailed,
		RC:          400,
		Action:      "Verify that a volume attachment for your instance exists. Run 'ibmcloud is in-vols INSTANCE_ID' to list active volume attachments for your instance ID. ",
	},"